bug(#320): fix filename undefined error when loading extesion with vscode version 1.94.0 and newer

ireznik opened the pull request:
this should resolve the issue that after an update of vscode (>= v1.94) the extension no longer is loading.

[origin of solution](https://github.com/robb0wen/synthwave-vscode/issues/326#issuecomment-2402593191)

Rottbott commented:
You'll need to make equivalent changes in `uninstall()` so the theme can be disabled cleanly.

ireznik commented:
> You'll need to make equivalent changes in `uninstall()` so the theme can be disabled cleanly.

thanks for the hint. I've updated it

karlnyr commented:
Deployed when? My code is missing the SynthWave vibes 😭

vesk4000 commented:
I just installed the theme and the Neon wasn't working, but this PR does indeed fix the issue. I hope to see it merged soon, because this theme is awesome!

callit commented:
This looks the cleanest out of the few PRs that are currently out there. Applied updates and both enable and disable work now. Thanks!

callit reviewed (approved):
These changes seem cleanest out of the other PRs that I've looked at and function as expected. I'd vote on this one being merged personally.

robb0wen commented:
I like this solution. My only concern is breaking backwards compatibility for people stuck on older versions of VS code. It might be worth looking at preserving the old filenames and paths behind a conditional with `isVSCodeBelowVersion("1.94.0")` or something to that effect

ireznik commented:
@robb0wen I've updated the changes with your suggestion. I also replaced all windows / linux paths in that particular area with path.join since this should not change behavior to make it easier to understand.

robb0wen commented:
Thanks again for your help on this one!
